recipes/repeatmodeler/RepModelConfig.pm

```diff
@@ -104,14 +104,14 @@ BEGIN
 ## ============
 ## Tandem Repeat Finder program.
 ##
-  $TRF_PRGM = $ENV{'TRF_DIR'};
-
+  $TRF_DIR = $ENV{'TRF_DIR'};
+  $TRF_PRGM = "$TRF_DIR/trf";
 ##
 ## NSEG Location
 ## =============
 ## Location of the NCBI nseg program
-  $NSEG_PRGM = $ENV{'NSEG_DIR'};
-
+  $NSEG_DIR = $ENV{'NSEG_DIR'};
+  $NSEG_PRGM = "$NSEG_DIR/nseg";
```

Comment:
If you depend on the `nseg` Bioconda package that you created yesterday, then `nseg` will be just installed into `$PATH` and it should not be necessary for this to include `$NSEG_DIR` (or actually to define a `$NSEG_DIR` at all).

Reply:
It's just to stay consistent with how the other paths are defined by the author in this file. And he define them in the `build.sh`. I don't know if it use elsewhere in the code...

Reply:
That said, I don’t know how this setting is used actually.